### PR TITLE
Documentation/design: add high-level workflow diagram

### PR DESCRIPTION
**Pull request by abhinavdahiya**
Add SVG for the high-level steps in installer.

### CODE REVIEW

File: Documentation/design/workflow.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# High-level Install Workflow Diagram
+
+![High-level Install Workflow diagram](./workflow.svg)
+
+<img src="./workflow.svg">
```
[the remark this answers is not in the document]
Comment (abhinavdahiya) [reply Documentation/design/workflow.md]: removed ``

**Comment by wking** [comment]
Can the SVG live in `Documentation/dev/node-bootstrap-flow.svg` and replace [this ASCII-art][1]?

[1]: https://github.com/openshift/installer/blob/ae41b0a3fa6f3c074bb1e0fd726a9aec406c4301/Documentation/dev/node-bootstrap-flow.md#diagram

**Comment by abhinavdahiya** [comment]
@wking `Documentation/dev/node-bootstrap-flow.svg` is from old installer and might be updated soon. Workflow svg is installer workflow not just how bootstrap node works.

**Comment by abhinavdahiya** [comment]
@yifan-gu ci-bot need `/lgtm`

**Pull request merged**